pkg/collector/corechecks/snmp/internal/report/report_device_metadata.go

```diff
@@ -45,6 +48,42 @@ func (ms *MetricSender) ReportNetworkDeviceMetadata(config *checkconfig.CheckCon
 		}
 		ms.sender.EventPlatformEvent(string(payloadBytes), epforwarder.EventTypeNetworkDevicesMetadata)
 	}
+
+	// Telemetry
+	for _, interfaceStatus := range interfaces {
+		status := computeInterfaceStatus(interfaceStatus.AdminStatus, interfaceStatus.OperStatus)
+		interfaceTags := []string{"status:" + status, "interface:" + interfaceStatus.Name, "interface_alias:" + interfaceStatus.Alias}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
No need to add device id tags too here ?
What happens if we send two metrics for the same interface name, index from two different devices ?

Reply:
Since we are also appending the `tags` that contain all device related tags, I think that should be enough to disambiguate interfaces with same names/index.

The `tags` contain device ip and namespace.